sdk/core/azure_core/src/lib.rs

```diff
@@ -11,16 +11,13 @@
 mod macros;
 
 pub mod cloud;
-mod constants;
 pub mod credentials;
 pub mod error;
 pub mod hmac;
 pub mod http;
 #[cfg(feature = "test")]
 pub mod test;
 
-pub use constants::*;
```

Comment:
You need to update `CHANGELOG.md` for `azure_core`.

Reply:
I'm trying to prevent merge conflicts by waiting for the post-release update